prefetch.bs

```diff
@@ -863,6 +863,16 @@ The \`<code><dfn http-header>Sec-Speculation-Tags</dfn></code>\` HTTP request he
 
 The header field is an [[RFC9651]] Structured Header whose value must be a [=structured header/List=]. The list can contain either [=structured header/Token=] or [=structured header/String=] values. String values represent developer-provided tags, whereas token values represent predefined tags. As of now, the only predefined tag is `null`, which indicates a speculative navigation request with no developer-defined tag.
 
+<h2 id="interaction-with-other-specs">Interaction with other specifications and concepts</h2>
+
+<div algorithm="clear prefetch cache for origin patch">
+  Modify [$clear cache for origin$] to add the following step at the end.
+
+  1. [=list/For each=] [=prefetch record=] |prefetchRecord| in the user agent's |activeDocument|'s [=Document/prefetch records=]:
+  1. If |prefetchRecord|'s [=document state/initiator origin=] is [=same origin=] with |activeDocument|'s [=Document/origin=], then [=prefetch record/cancel and discard=] |prefetchRecord| given <var ignore>origin</var>.
```

Comment:
This should be indented so that it's nested in the output. (I realize we didn't catch this on prerendering... I'll push a quick commit fixing it.)

Reply:
Thank you!